cipher-symmetric/src/main/scala/tsec/cipher/symmetric/imports/package.scala

```diff
@@ -12,13 +12,13 @@ package object imports{
   type AEADCipherText[A] = CipherText[A, GCM, NoPadding]
 
   /**
-   * Typeclass for propagating symmetric key algorithm information
-   * Note: Key length is in bits
-   *
-   * @param algorithm the symmetric cipher representation, as a string
-   * @param keyLength key length in bits
-   * @tparam T Parametrized cipher type
-   */
+    * Typeclass for propagating symmetric key algorithm information
```

Comment:
Java or Scala Comment Blocks? I've seen this switch more then once now I think.

Reply:
@ChristopherDavenport I didn't know the difference, is it _only_ the first line, scala starts with /** Typeclass .... ?